cloudify/utils.py

```diff
@@ -266,6 +266,45 @@ def get_execution_token():
         return None
 
 
+def get_instances_of_node(deployment_id=None, node_id=None, _ctx=None):
+    """
+    Get node instances of a node from a deployment.
+    """
+    _ctx = _ctx or _get_current_context()
+    deployment_id = deployment_id or _ctx.deployment.id
+    node_id = node_id or _ctx.node.id
+    from cloudify.manager import get_rest_client
+    rest_client = get_rest_client()
```

Comment:
@EarthmanT what about passing the rest client as part of the method signature instead of using import inside the body of this method ?, so I'm pretty sure that rest client is importing the `utils` and the way you put the import here in order to avoid the circular dependency